CRM/Contribute/Selector/Search.php

```diff
@@ -496,7 +504,7 @@ public function &getColumnHeaders($action = NULL, $output = NULL) {
     $pre = [];
     self::$_columnHeaders = [
       [
-        'name' => $this->_includeSoftCredits ? ts('Contribution Amount') : ts('Amount'),
+        'name' => ($this->_includeSoftCredits ? ts('Contribution Amount') : ts('Amount')) . ' (due)',
```

Comment:
The concatenation here seems a bit odd, and is missing `ts`.

If the `(due)` is always to be displayed, could you add it to both strings, ex: `ts('Contribution Amount (due)')`  ?